setupext.py

```diff
@@ -252,29 +253,15 @@ def __init__(self):
         if sys.platform == 'win32':
             self.has_pkgconfig = False
         else:
-            self.set_pkgconfig_path()
-            status, output = getstatusoutput("pkg-config --help")
+            self.pkgconfig = os.environ.get('PKG_CONFIG', 'pkg-config')
+            status, output = getstatusoutput("%s --help" % self.pkgconfig)
             self.has_pkgconfig = (status == 0)
             if not self.has_pkgconfig:
                 print("IMPORTANT WARNING:")
                 print(
                     "    pkg-config is not installed.\n"
                     "    matplotlib may not be able to find some of its dependencies")
 
-    def set_pkgconfig_path(self):
```

Comment:
Why was this removed?
Note that (unchecked...) this *may* be needed to make things work in a conda env which does install the libraries in the environment but where pkg-config is "outside" the env (because sysconfig.get_config_var("LIBDIR") will correctly point inside the env in that case).